packages/consumption/src/modules/settings/SettingsController.ts

```diff
@@ -56,4 +56,11 @@ export class SettingsController extends ConsumptionBaseController {
     public async deleteSetting(setting: Setting): Promise<void> {
         await this.settings.delete(setting);
     }
+
+    public async deleteSettingsForRelationship(relationshipId: CoreId): Promise<void> {
+        const settings = await this.getSettings({ reference: relationshipId.toString(), scope: SettingScope.Relationship });
+        for (const setting of settings) {
+            await this.deleteSetting(setting);
```

Comment:
why this instead of Promise.all ? (similarly for relationship templates and tokens)

Reply:
don't want to parallelize db calls espacially if the datawallet is used